Content.Server/Botany/SeedPrototype.cs

```diff
@@ -126,14 +126,14 @@ public class SeedData
 
     [DataField("waterConsumption")] public float WaterConsumption = 3f;
     [DataField("idealHeat")] public float IdealHeat = 293f;
-    [DataField("heatTolerance")] public float HeatTolerance = 20f;
+    [DataField("heatTolerance")] public float HeatTolerance = 10f;
```

Comment:
What's with all these changes to default tolerances?

Reply:
I explained this in the PR description: Adjust the default tolerances to be a bit more reasonable.

Basically, if these weren't changed, mutations across these tolerances would not be noticable, i.e. the plant was too tolerant to heat to begin with, so if the heat gene was mutated, you'd see nothing happen.